cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/LoadMonitorState.java

```diff
@@ -200,28 +232,30 @@ public static LoadMonitorState loading(int numValidSnapshotWindows,
    * round it to zero when Java Math sees a NaN
    */
   public static double nanToZero(double v) {
-      if (Double.isNaN(v)) {
-          return 0.0;
-      } else {
-          return v;
-      }
+      return Double.isNaN(v) ? 0.0 : v;
   }
 
 
   @Override
   public String toString() {
     String trained = ModelParameters.trainingCompleted()
-        ? "(TRAINED)" : String.format("(%.3f%% trained)",
-                                      ModelParameters.modelCoefficientTrainingCompleteness() * 100);
+                     ? "(TRAINED)"
+                     : String.format("(%.3f%% trained)", ModelParameters.modelCoefficientTrainingCompleteness() * 100);
     float validPartitionPercent = (float) _numValidMonitoredPartitions / _totalNumPartitions;
     float validWindowPercent = (float) _numValidWindows / _monitoredWindows.size();
     switch (_loadMonitorTaskRunnerState) {
       case NOT_STARTED:
         return String.format("{state: %s}", _loadMonitorTaskRunnerState);
 
       case RUNNING:
-      case SAMPLING:
       case PAUSED:
+        return String.format("{state: %s%s, NumValidWindows: (%d/%d) (%.3f%%), NumValidPartitions: %d/%d (%.3f%%), flawedPartitions: %d%s}",
```

Comment:
missing the format field for `_reasonOfLatestPauseOrResume`

Reply:
The `%s` at the end is used for `_reasonOfLatestPauseOrResume`.